packages/wonder-blocks-modal/components/one-column-modal.js

```diff
@@ -30,7 +30,7 @@ type BaseProps = {|
      * clicks too.)
      * @ignore
      */
-    onClickCloseButton?: () => void,
+    onClose?: () => void,
```

Comment:
It looks like this can be made mandatory now.

Also, should this property still be marked with "@ignore"? (That made sense before when we were passing it through but less so now if you're expecting people to use this property directly.)